server/main.go

```diff
@@ -126,6 +128,12 @@ func main() {
 			Usage:       "mongo database name",
 			Destination: &dbName,
 		},
+		cli.StringFlag{
+			Name:        "config-file, config",
+			Value:       "./settings.json",
```

Comment:
There shouldn't be a default value, and this file should be entirely optional - everything else should still work when the file is missing or empty.

Would be good to support an environment variable too.

If we want this to be a generic settings/config file, then we should change the format since it only has signers right now. Otherwise, this flag name should be something like `signers`.

Reply:
Yeah as we discussed separately I'll rename to signers